examples/golden_graph/graph.hpp

```diff
@@ -0,0 +1,16 @@
+#ifndef EXAMPLES_GOLDEN_GRAPH_GRAPH_HPP
+#define EXAMPLES_GOLDEN_GRAPH_GRAPH_HPP
+
+#include <vector>
+
+class Graph
+{
+private:
+    bool _checkID(int id);
+    std::vector<Node*> _nodes;
```

Comment:
add getter function to return vector of nodes

Reply:
done